### ISSUE TITLE
1.82 clock_gettime build failure on macOS 10.12 Sierra

### ISSUE THREAD

**Issue by ilovezfs** [s3fs-fuse/s3fs-fuse#600]
Version 1.82 (and 1.81) fails to build on macOS 10.12 Sierra with the following error:
```
cache.cpp:62:10: error: no matching function for call to 'clock_gettime'
  return clock_gettime(clk_id, ts);
         ^~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include/time.h:177:5: note: candidate function not viable: no known conversion from 'int' to 'clockid_t' for 1st argument
int clock_gettime(clockid_t __clock_id, struct timespec *__tp);
    ^
```

Full build log is here: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1325/version=sierra/testReport/junit/brew-test-bot/sierra/install_s3fs/

**Comment by ggtakec** [comment]
@ilovezfs Fixed this problem by #611.
Please try to check this.
And Travis CI setting change is tracked with #601.

**Comment by ilovezfs** [comment]
@ggtakec fix looks good:

Before: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2078/version=sierra/console

After: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2079/version=sierra/console

**Comment by ggtakec** [comment]
@ilovezfs Thank you for test and reply.
If you have a problem about s3fs, please post new issue.
Regards,

**Comment by ilovezfs** [comment]
You're welcome!
